lib/apiservers/engine/backends/backends.go

```diff
@@ -84,7 +94,15 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 			portLayerName = product + " " + productVersion + " Backend Engine"
 		}
 
-		parseFriendlyRegistryString(config) // For docker info
+		var err error
+		if vchConfig.Insecure, err = dynamic.ParseRegistries(config.InsecureRegistries); err != nil {
```

Comment:
Right now this is not reaching out to the OVA. My plan was to do that on demand, and caching that lookup info for subsequent use (until it no longer works).